Composer/packages/client/src/constants/colors.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getTheme } from 'office-ui-fabric-react/lib/Styling';
+
+const theme = getTheme();
+
+const semColors = theme.semanticColors;
+
+export const colors = {
+  // copied from https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/messaging
+  errorBg: semColors.errorBackground,
```

Comment:
nit: I usually prefer expanded names. Bg and Fg im assuming are Background and foreground.